pymeasure/instruments/keithley/keithley2200.py

```diff
@@ -23,129 +23,93 @@
         map_values=True
     )
 
     voltage = Instrument.control(
         "VOLT?", "VOLT %g",
-        """ A floating point property that represents the output voltage
-        setting of the power supply in Volts. This property can be set. """,
+        """ A floating point property representing the output voltage in Volts.""",
         validator=strict_range,
         values=VOLTAGE_RANGE
     )
 
     current = Instrument.control(
         "CURR?", "CURR %g",
-        """ A floating point property that represents the output current of
-        the power supply in Amps. This property can be set. """,
+        """ A floating point property that representing the output current in Amps.""",
         validator=strict_range,
         values=CURRENT_RANGE
     )
 
     measure_current = Instrument.measurement(
         "MEAS:CURR?",
-        """ A measurement property that reads the measured current in Amps.
-        This property is readonly. """,
+        """ A measurement property that reads the measured current in Amps.""",
     )
 
     measure_voltage = Instrument.measurement(
         "MEAS:VOLT?",
-        """ A measurement property that reads the measured current in Volts.
-        This property is readonly. """,
+        """ A measurement property that reads the measured current in Volts.""",
     )
 
     measure_power = Instrument.measurement(
         "MEAS:VOLT?",
-        """ A measurement property that reads the measured power in watts.
-        This property is readonly. """,
+        """ A measurement property that reads the measured power in watts.""",
     )
 
     voltage_limit = Instrument.control(
         "VOLT:LIM?", "VOLT:LIM %g",
-        """ A property which represents the maximum voltage limit""",
+        """ A property which represents the maximum voltage limit.""",
         validator=strict_range,
         values=VOLTAGE_RANGE
     )
 
     voltage_limit_enabled = Instrument.control(
         "VOLT:LIM:STAT?", "VOLT:LIM:STAT %d",
-        """A boolean property that controls whether the voltage limit is enabled,
-        takes values True or False. """,
+        """A boolean property that representing voltage limit.""",
         validator=strict_discrete_set,
         values={True: 1, False: 0},
         map_values=True
     )
 
-    def __init__(self, instrument, number):
-        self.instrument = instrument
-        self.number = number
-
-    def values(self, command, **kwargs):
-        return self.instrument.values(f"INST:SEL CH{self.number};{command}",
-                                      **kwargs)
-
-    def write(self, command):
-        self.instrument.write(f"INST:SEL CH{self.number};{command}")
+    def insert_id(self, command):
+        return f"INST:SEL CH{self.id};{command}"
 
 
 class Keithley2200(Instrument):
     """ Represents the Keithley 2230 Power Supply.
     """
+    def __init__(self, adapter, name="Keithley2200"):
+        super().__init__(adapter, name)
```

Comment:
Please keep the kwargs in:
```suggestion
    def __init__(self, adapter, name="Keithley2200", **kwargs):
        super().__init__(adapter, name, **kwargs)
```

Reply:
fixed in commit

Reply:
Did you push that commit, because they are not in the current version online.

Reply:
my mistake, pushed the commit.